tools/cargos/src/main.rs

```diff
@@ -0,0 +1,253 @@
+// Copyright 2023 The Fuchsia Authors
+//
+// Licensed under a BSD-style license <LICENSE-BSD>, Apache License, Version 2.0
+// <LICENSE-APACHE or https://www.apache.org/licenses/LICENSE-2.0>, or the MIT
+// license <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your option.
+// This file may not be copied, modified, or distributed except according to
+// those terms.
+
+// This script is a thin wrapper around Cargo that provides human-friendly
+// toolchain names which are automatically translated to the toolchain versions
+// we have pinned in CI.
+//
+//   cargos --version <toolchain-name> # looks up the version for the named toolchain
+//   cargos +<toolchain-name> [...]    # runs cargo commands with the named toolchain
+//   cargos +all [...]                 # runs cargo commands with each toolchain
+//
+// The meta-toolchain "all" instructs this script to run the provided command
+// once for each "major" toolchain (msrv, stable, nightly). This does not
+// include any toolchain which is listed in the `package.metadata.build-rs`
+// Cargo.toml section.
+//
+// A common task that is especially annoying to perform by hand is to update
+// trybuild's stderr files. Using this script:
+//
+//   TRYBUILD=overwrite ./cargos +all test --workspace
+
+use std::{
+    env::{args, vars},
```

Comment:
Uber nit: Can you just `use env` and then invoke these as `env::args()` and `env::vars()` to be consistent with our existing style?

Reply:
Done